org.wxhexeditor.wxHexEditor.json

```diff
@@ -0,0 +1,73 @@
+{
+    "app-id": "org.wxhexeditor.wxHexEditor",
+    "only-arches": ["x86_64", "i386"],
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "wxHexEditor",
+    "rename-icon": "wxHexEditor",
+    "rename-desktop-file": "wxHexEditor.desktop",
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
```

Comment:
Does this actually support Wayland?

Reply:
It uses gtk3 as its backend. (wxgtk3)

Reply:
Using gtk3 alone does not mean it works on Wayland.

Reply:
someone else will probably have to test that, as my notebook stubbornly refuses to start gnome-wayland

Reply:
It behaves very weird under wayland.
It works (kind of) but it does not update the display.
This seems like an upstream (maybe even wxWidgets) problem.

Reply:
Disabled for now.
This is jet another upstream issue.